pom.xml

```diff
@@ -1769,6 +1769,7 @@
                 <module>tests/ballerina-tools-integration-test</module>
                 <module>tests/ballerina-compiler-plugin-test</module>
                 <module>tests/ballerina-examples-test</module>
+                <module>spec-conformance-tests</module>
```

Comment:
Is there any specific reason for this to be added to root level? All other tests are in the `tests` directory. So this looks a bit odd.

Reply:
+1, shall we move this under tests ?